node/src/main/scala/coop/rchain/node/runtime/NodeRuntime.scala

```diff
@@ -405,29 +404,6 @@ class NodeRuntime[F[_]: Monixable: ConcurrentEffect: Parallel: Timer: ContextShi
     } yield ()
   }
 
-  def addShutdownHook(
-      servers: ServersInstances[F],
-      runtimeCleanup: Cleanup[F],
-      blockStore: BlockStore[F]
-  ): F[Unit] =
-    Sync[F].delay(sys.addShutdownHook(clearResources(servers, runtimeCleanup, blockStore))).void
-
-  def clearResources(
-      servers: ServersInstances[F],
-      runtimeCleanup: Cleanup[F],
-      blockStore: BlockStore[F]
-  ): Unit = {
-    val shutdown = for {
-      _ <- Sync[F].delay(Kamon.stopAllReporters())
-      _ <- runtimeCleanup.close()
-      _ <- Log[F].info("Bringing BlockStore down ...")
-      _ <- blockStore.close()
```

Comment:
blockstore is now managed by node store manager, so no need to close it separately

Reply:
We didn't migrated old full nodes to LMDB based block store.